core/includes/file.inc

```diff
@@ -536,6 +536,9 @@ function file_save_htaccess($directory, $private = TRUE, $force_overwrite = FALS
  */
 function file_htaccess_lines($private = TRUE) {
   $lines = <<<EOF
+# This code block is replicated in the file_htaccess_lines() function in
+# /core/includes/file.inc and files/.htaccess. If you update this file then 
+# make sure to also update the other location accordingly.
 # Turn off all options we don't need.
```

Comment:
Can we please add a blank line above this one, in order to separate the comment that explains the code from the one re the replication of the files?